pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/JsonMailboxIdentifier.java

```diff
@@ -106,6 +116,16 @@ public boolean isLocal() {
     return _fromAddress.equals(_toAddress);
   }
 
+  @Override
```

Comment:
self-review: Add JsonIgnore annotations and make getters order same as declarations.